xbmc/cores/VideoPlayer/DVDSubtitles/SubtitleParserWebVTT.cpp

```diff
@@ -32,19 +32,36 @@ bool CSubtitleParserWebVTT::Open(CDVDStreamInfo& hints)
   if (!Initialize())
     return false;
 
+  CLog::LogF(LOGINFO, "Initialized WebVTT Parser");
```

Comment:
please remove all these logs here, are not useful and are also inconsistent with overlay codec side